### PR TITLE
Vector division broken in python 3.x #1914

### PR DESCRIPTION
**Pull request by platinhom**
This pull request addresses issue #1914 

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
Looks good - I'll allow a few days in case @etal or any of the other PDB experts wants to comment, but hope to merge this during the coming week.

**Comment by platinhom** [comment]
I'm not familiar with the github pull requests and the code check 😑

**Comment by peterjc** [comment]
@platinhom it was new to me once, and Github improves it over time - so I keep learning by doing.

File: Bio/PDB/vectors.py [hunk 1]
```diff
@@ -281,6 +281,8 @@ def __div__(self, x):
         a = self._ar / numpy.array(x)
         return Vector(a)
 
+    __truediv__ = __div__
```
Comment (lennax) [inline Bio/PDB/vectors.py]: What if we do the opposite? Rename the current method to `__truediv__` and assign `__div__ = __truediv__`. That way, when we drop 2.x support, we can just remove `__div__`.

File: Tests/test_PDB_vectors.py [hunk 2]
```diff
@@ -36,6 +36,16 @@ def test_legacy_imports(self):
...
 
 
+class VectorTests(unittest.TestCase):
+    """Tests for the Vector class."""
+
+    def test_division(self):
+        """Confirm division works."""
+        from Bio.PDB.Vector import Vector
+        v = Vector(2, 2, 2) / 2
+        self.assertEqual(repr(v), "<Vector 1.00, 1.00, 1.00>")
```
Comment (lennax) [inline Tests/test_PDB_vectors.py]: I'm probably being too picky but this test wouldn't fail if the vector was somehow divided using floor division then cast to int. I'd use example numbers that would look even more strange if floor division were used (e.g. the classic `1 // 2 == 0`).

Comment (peterjc) [reply Tests/test_PDB_vectors.py]: I wondered that too - this would be clearer, good idea.

File: Bio/PDB/vectors.py (hunk 1 above)
Comment (peterjc) [reply Bio/PDB/vectors.py]: That seems a worthwhile improvement.

**Comment by peterjc** [comment]
Sorry, I would have merged this as is, but if you're willing to polish this further based on @lennax's useful comments, that'd be even better.

**Comment by peterjc** [comment]
The tests passed, but GitHub seems a bit slow to show that. Odd.

**Comment by peterjc** [comment]
Changes applied - thank you both!

**Pull request merged**